db/migrate/20220114231638_create_comments.rb

```diff
@@ -0,0 +1,5 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments, &:timestamps
```

Comment:
- [ ] We are missing the references to users table and references to table posts, actually we need to know which post is the comment referring to, and who wrote that comment, that's why you need foreign key to table users and the other one to table comments
- [ ] And also table comments need to have a text field, remember to add it

Reply:
Hi Descholor,
Thank you for the review I had forgotten to save files after including Foreign keys and when I push some files was empty but I fix the issue

Reply:
I see and thank you for the reply @BenMukebo , We are still missing this comment to be addressed correctly: `The comments table should have a _text_ field, so that every comment can have its content`